docs/LayoutWithFlexbox.md

```diff
@@ -1,10 +1,10 @@
 ---
-id: basics-layout
+id: layout-with-flexbox
 title: Layout with Flexbox
 layout: docs
 category: The Basics
 permalink: docs/flexbox.html
```

Comment:
(I want to keep the permanlink as docs/flexbox.html here to maintain old links, bc there were a lot here)